upi/aws/cloudformation/02_cluster_infra.yaml

```diff
@@ -126,7 +126,7 @@ Resources:
       - Name:
           !Join [
             ".",
-            ["api", !Ref ClusterName, !Join ["", [!Ref HostedZoneName, "."]]],
+            ["api-int", !Ref ClusterName, !Join ["", [!Ref HostedZoneName, "."]]],
```

Comment:
Per my testing result, both 'api' and 'api-int' are required.

Reply:
Everything inside the cluster should use api-int. Are you sure you picked up [the][1] [other][2] related changes when testing?

[1]: https://github.com/openshift/installer/commit/79b05ec6395c2168ddcf124f8b31750a67498cca#diff-a8b48e638840fba90403297f334dfd95L41
[2]: https://github.com/openshift/installer/commit/79b05ec6395c2168ddcf124f8b31750a67498cca#diff-7aaf8ee54fe5d87f7a10c87052d253caL22

Reply:
Yeah, based on your PR is not merged yet, I copied the change into my own [branch](https://github.com/jianlinliu/upi-on-aws-cf-template). After I hit failure, I submitted [some fix](https://github.com/jianlinliu/upi-on-aws-cf-template/commits/master/02_cluster_infra.yaml), and confirm my fix works well.

Reply:
Ah, let me check my local shell script if update accordingly.

Reply:
Yeah, my fault, forget to change IgnitionLocation url accordingly. Ignore this comment.

Reply:
Hmm, seem using 'api-int' would hit x509 issue. Refer to https://bugzilla.redhat.com/show_bug.cgi?id=1697968#c13 more details.